datumaro/cli/contexts/project/__init__.py

```diff
@@ -8,8 +8,8 @@
 import os
 import os.path as osp
 import shutil
-import numpy as np
 from enum import Enum
```

Comment:
There is no such in PyLint, but please put `from x import y` before `import z`

Reply:
I suggest that we stick to Pylint issues in this PR. We can use `isort` to reorder all imports automatically later.